cloud/scope/machine.go

```diff
@@ -78,6 +81,14 @@ func NewMachineScope(params MachineScopeParams, authenticator core.Authenticator
 		return nil, errors.Wrap(err, "failed to init patch helper")
 	}
 
+	// Create the service endpoint.
+	svcEndpoint := "https://" + params.IBMVPCCluster.Spec.Region + ".iaas.cloud.ibm.com/v1"
+	for _, vpcEndpoint := range params.ServiceEndpoint {
+		if vpcEndpoint.ServiceRegion == params.IBMVPCCluster.Spec.Region && vpcEndpoint.ServiceID == endpoints.VPCServiceID {
+			svcEndpoint = vpcEndpoint.URL + "/v1"
```

Comment:
why don't we expect user to mention "/v1" to user to keep it simple?

Reply:
We can expect it from the user, (I was of the opinion to keep the url simple for user and append that in our code).

Reply:
Its always good to stick with the endpoints provided in the doc, e.g:
<img width="580" alt="image" src="https://user-images.githubusercontent.com/12646029/172556301-9812c2d3-3f4e-4e8d-b426-0d875d8096b1.png">